testutil/fuzz.go

```diff
@@ -104,38 +104,58 @@ func NewEth2Fuzzer(t *testing.T, seed int64) *fuzz.Fuzzer {
 					*e = (*e)[:4]
 				}
 			},
-			// Populate one of the versions of these Versioned*Block types.
-			func(e *core.VersionedSignedBlindedBeaconBlock, c fuzz.Continue) {
+			// Populate one of the versions of these Versioned*Proposal types.
+			func(e *core.VersionedSignedBlindedProposal, c fuzz.Continue) {
 				e.Version = blindedVersions[(c.Intn(len(blindedVersions)))]
 				version, err := eth2util.DataVersionFromETH2(e.Version)
 				require.NoError(t, err)
 
 				val := core.VersionedSSZValueForT(t, e, version)
 				c.Fuzz(val)
+
+				maxBlobSidecars := 6 // Limit length of blob sidecars to 6
+				if e.Version == eth2spec.DataVersionDeneb && len(e.Deneb.SignedBlindedBlobSidecars) > maxBlobSidecars {
```

Comment:
Could we abstract this in a function?

Reply:
yes can be done, but I don't think it's worth it introducing a function for it. Moreover, there are 4 different types to deal with: Deneb.SignedBlindedBlobSidecars, Deneb.BlindedBlobSidecars, Deneb.SignedBlobSidecars, Deneb.BlobSidecars